scripts/add_nonce.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+# This must be called from repo root as current directory
+
+NONCE=$(date +%s)
+
+for packagedir in ./packages/*; do
+  if [[ -d "$packagedir" ]]; then
+    echo "$packagedir"
```

Comment:
Would be nice to expand a little. Suggest `"Added nonce to $packagedir" and putting it at the end of the loop.

Reply:
Good point, this is an old debugging log